packages/contracts/src/dollar/libraries/LibUbiquityPool.sol

```diff
@@ -0,0 +1,345 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.19;
+
+// Modified from FraxPool.sol by Frax Finance
+// https://github.com/FraxFinance/frax-solidity/blob/master/src/hardhat/contracts/Frax/Pools/FraxPool.sol
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {UbiquityDollarToken} from "../core/UbiquityDollarToken.sol";
+import {IERC20Metadata} from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {IERC20Ubiquity} from "../interfaces/IERC20Ubiquity.sol";
+import {SafeMath} from "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import {IStableSwap3Pool} from "../interfaces/IStableSwap3Pool.sol";
+import {IMetaPool} from "../interfaces/IMetaPool.sol";
+import {LibAppStorage, AppStorage} from "./LibAppStorage.sol";
+import {LibTWAPOracle} from "./LibTWAPOracle.sol";
+
+library LibUbiquityPool {
+    using SafeMath for uint256;
+    using SafeMath for uint8;
+    using SafeERC20 for IERC20;
+
+    bytes32 constant UBIQUITY_POOL_STORAGE_POSITION =
+        keccak256("ubiquity.contracts.ubiquity.pool.storage");
```

Comment:
The `ubiquity.contracts.ubiquity` prefix seems redundant.

Reply:
we need a unique name for each contract's storage position to avoid collisions

Reply:
I understand this. I still stand behind my statement.